docs/language/src/builtins/enums.md

```diff
@@ -75,14 +77,20 @@ The default value of each enum type is always the first value.
 
 ## `LayoutAlignment`
 
- Enum representing the alignment property of a BoxLayout or HorizontalLayout
-
-* **`stretch`**:
-* **`center`**:
-* **`start`**:
-* **`end`**:
-* **`space-between`**:
-* **`space-around`**:
+ Enum representing the alignment property of a BoxLayout or HorizontalLayout.
+ See [alignment](concepts/layouting.md#alignment)
+
+* **`stretch`**: Use the minimum size of all elements in a layout, distribute remaining space
+    based on `*-stretch` among all elements.
+* **`center`**: Use minimum size for all elements, distribute remaining space evenly before the
+    first and after the last element
+* **`start`**: Use minimum size for all elements, put remaining space after the last element
```

Comment:
```suggestion
* **`start`**: Use minimum size for all elements, put remaining space after the last element.
```

Reply:
Are you sure that it's the minimum size that's used? I thought it's the preferred size. @ogoffart  ?

Same goes for center.

AFAICS it's the preferred size that's used: [testcase](https://slint-ui.com/releases/0.3.5/editor/?snippet=import+%7B+VerticalBox+%7D+from+%22std-widgets.slint%22%3B%0Aexport+component+Demo+%7B%0A++++VerticalBox+%7B%0A++++++++alignment%3A+center%3B%0A++++++++Rectangle+%7B%0A++++++++++++background%3A+green%3B%0A++++++++++++min-height%3A+20px%3B%0A++++++++++++preferred-height%3A+100px%3B%0A++++++++%7D%0A++++++++Rectangle+%7B%0A++++++++++++background%3A+red%3B%0A++++++++++++min-height%3A+30px%3B%0A++++++++%7D%0A++++%7D%0A%7D%0A)